packages/dbos-cloud/cloudutils.ts

```diff
@@ -15,15 +15,17 @@ export const dbosConfigFilePath = "dbos-config.yaml";
 export const DBOSCloudHost = process.env.DBOS_DOMAIN || "cloud.dbos.dev";
 export const dbosEnvPath = ".dbos";
 
-export function retrieveApplicationName(logger: Logger): string | null {
+export function retrieveApplicationName(logger: Logger, silent: Boolean = false): string | null {
```

Comment:
Maybe the calling code should do the logging? passing in the logger and a parameter to control when to log seems a little odd